backend/b2/b2_internal_test.go

```diff
@@ -170,9 +185,234 @@ func TestParseTimeString(t *testing.T) {
 
 }
 
+// This is adapted from the s3 equivalent.
+func (f *Fs) InternalTestMetadata(t *testing.T) {
```

Comment:
This looks like it doesn't belong in this commit?

Reply:
I've backed it out of this PR.